mods/translucent-windows.wh.cpp

```diff
@@ -515,6 +562,77 @@ BOOL IsWindowClass(HWND hWnd, LPCWSTR ClassName)
     return FALSE;
 }
 
+void UnsubclassWindow(HWND hWnd)
+{
+    WindhawkUtils::RemoveWindowSubclassFromAnyThread(hWnd, SubclassProc);
+
+    std::lock_guard<std::mutex> guard(g_subclassedWindowsMutex);
+
+    auto it = g_subclassedWindows.find(hWnd);
+    if (it != g_subclassedWindows.end()) {
+        g_subclassedWindows.erase(it);
+    }
+}
+
+LRESULT CALLBACK SubclassProc(HWND hWnd, UINT uMsg, WPARAM wParam, LPARAM lParam, DWORD_PTR dwRefData)
+{
+    switch (uMsg)
+    {
+        case WM_ACTIVATE:
+        {
+            bool isMinimized = HIWORD(wParam);
+            if (!isMinimized)
+            {
+                WORD activationState = LOWORD(wParam);
+
+                if ((activationState == WA_ACTIVE || activationState == WA_CLICKACTIVE))
+                {
+                    if(g_settings.TitlebarFlag)
+                    {
+                        g_settings.g_TitlebarColor = g_settings.TitlebarActiveColor;
+                        DwmSetWindowAttribute(hWnd, CAPTION_COLOR, &g_settings.g_TitlebarColor, sizeof(g_settings.g_TitlebarColor));
+                    }
+                    if(g_settings.CaptionTextFlag)
+                    {
+                        g_settings.g_CaptionColor = g_settings.CaptionActiveTextColor;
+                        DwmSetWindowAttribute(hWnd, CAPTION_TEXT_COLOR, &g_settings.g_CaptionColor, sizeof(g_settings.g_CaptionColor));
+                    }
+                    if(g_settings.BorderFlag)
+                    {
+                        g_settings.g_BorderColor = g_settings.BorderActiveColor;
+                        DwmSetWindowAttribute(hWnd, BORDER_COLOR, &g_settings.BorderActiveColor, sizeof(g_settings.BorderActiveColor));
+                    }
+                }
+                else if (activationState == WA_INACTIVE)
+                {
+                    if(g_settings.TitlebarFlag)
+                    {
+                        g_settings.g_TitlebarColor = g_settings.TitlebarInactiveColor;
+                        DwmSetWindowAttribute(hWnd, CAPTION_COLOR, &g_settings.g_TitlebarColor, sizeof(g_settings.g_TitlebarColor));
+                    }
+                    if(g_settings.CaptionTextFlag)
+                    {
+                        g_settings.g_CaptionColor = g_settings.CaptionInactiveTextColor;
+                        DwmSetWindowAttribute(hWnd, CAPTION_TEXT_COLOR, &g_settings.g_CaptionColor, sizeof(g_settings.g_CaptionColor));
+                    }
+                    if(g_settings.BorderFlag)
+                    {
+                        g_settings.g_BorderColor = g_settings.BorderInactiveColor;
+                        DwmSetWindowAttribute(hWnd, BORDER_COLOR, &g_settings.BorderInactiveColor, sizeof(g_settings.BorderInactiveColor));
+                    }
+                }
+            }
+            break;
+        }
+        case WM_NCDESTROY:
+        {
+            UnsubclassWindow(hWnd);

```

Comment:
No need to unsubclass here, it's enough to do:
```suggestion
            std::lock_guard<std::mutex> guard(g_subclassedWindowsMutex);
            g_subclassedWindows.erase(hWnd);
```